user_stories.md

```diff
@@ -0,0 +1,40 @@
+Methodology...
```

Comment:
User stories follow the format of “as a ___ I want to ___ so that ___” For example, As a marketing researcher, I want to keep track of the images the user clicks on so that I can determine which items are the most popular.